yb-voyager/src/importdata/errorHandlers.go

```diff
@@ -108,28 +114,40 @@ func (handler *ImportDataStashAndContinueHandler) HandleBatchIngestionError(batc
 // create a symlink to the batch file in the errors folder so that all errors are in one place.
 // ingestion-error.<batch_file_name> -> metainfo/.../<batch_file_name>
 func (handler *ImportDataStashAndContinueHandler) createBatchSymlinkInErrorsFolder(batch ErroredBatch, taskFilePath string) error {
-	err := os.MkdirAll(handler.getErrorsFolderPathForTableTask(batch, taskFilePath), os.ModePerm)
+	err := os.MkdirAll(handler.getErrorsFolderPathForTableTask(batch.GetTableName(), taskFilePath), os.ModePerm)
```

Comment:
nit: we can store the result of `handler.getErrorsFolderPathForTableTask(batch.GetTableName(), taskFilePath)` and then reuse for the both the things.